testing/test_base.sh

```diff
@@ -5,9 +5,10 @@ source testing/test_preamble.sh
 echo Base Tests >> $TEST_RESULTS
 
 function redact {
-    sed -e 's/\s*%%.*//' \
-        -e 's/2019-.*T.*Z/XXX/' \
-        -e 's/2019-.*00:00/XXX/' \
+    sed -e '/^%%.*/d' \
+	-e 's/\s*%%.*//' \
+        -e 's/2020-.*T.*Z/XXX/' \
```

Comment:
These were fixed in master some time ago. Please rebase, resolve merge conflicts then commit.

Reply:
Resolved conflicts in rebase. Added new line which deletes log packet lines instead of replacing them with blank lines when comparing the redacted files, as this causes travis to fail on differing spaces. 

```
function redact {
    sed -E -e 's/\s*%%.*//' \
	-e '/^%%.*/d' \
        -e 's/[0-9]{4}-.*T.*Z/XXX/' \
        -e 's/[0-9]{4}-(0|1)[0-9]-(0|1|2|3)[0-9] [0-9]{2}:[0-9]{2}:[0-9]{2}\+00:00/XXX/g' \
        -e 's/DAQ version.*//'
}
```